docs/samples/Microsoft.ML.Samples/Dynamic/SsaChangePointDetectorTransform.cs

```diff
@@ -66,23 +75,127 @@ public static void SsaChangePointDetectorTransform()
-            // 5       0       0.00    0.50    0.00
-            // 7       1       2.00    0.00    10298.67   <-- alert is on, predicted changepoint
-            // 7       0       1.00    0.31    15741.58
-            // 7       0       0.00    0.28    26487.48
-            // 7       0       0.00    0.28    44569.02
-            // 7       0       0.00    0.28    0.01
-            // 7       0       0.00    0.38    0.01
-            // 7       0       0.00    0.50    0.00
-            // 7       0       0.00    0.50    0.00
+            // Data    Alert   Score   P-Value Martingale value
+            // 0       0     - 2.53    0.50    0.00
+            // 1       0     - 0.01    0.01    0.00
+            // 2       0       0.76    0.14    0.00
+            // 3       0       0.69    0.28    0.00
+            // 4       0       1.44    0.18    0.00
+            // 0       0     - 1.84    0.17    0.00
+            // 1       0       0.22    0.44    0.00
+            // 2       0       0.20    0.45    0.00
+            // 3       0       0.16    0.47    0.00
+            // 4       0       1.33    0.18    0.00
+            // 0       0     - 1.79    0.07    0.00
+            // 1       0       0.16    0.50    0.00
+            // 2       0       0.09    0.50    0.00
+            // 3       0       0.08    0.45    0.00
+            // 4       0       1.31    0.12    0.00
+            // 0       0     - 1.79    0.07    0.00
+            // 100     1      99.16    0.00    4031.94     <-- alert is on, predicted changepoint
+            // 200     0     185.23    0.00    731260.87
+            // 300     0     270.40    0.01    3578470.47
+            // 400     0     357.11    0.03    45298370.86
+        }
+
+        // This example shows change point detection as above, but demonstrates how to train a model
+        // that can run predictions on streaming data, and how to persist the trained model and then re-load it.
+        public static void SsaChangePointDetectorPrediction()
+        {
+            // Create a new ML context, for ML.NET operations. It can be used for exception tracking and logging, 
+            // as well as the source of randomness.
+            var ml = new MLContext();
+
+            // Generate sample series data with a recurring pattern
+            const int SeasonalitySize = 5;
+            const int TrainingSeasons = 3;
+            const int TrainingSize = SeasonalitySize * TrainingSeasons;
+            var data = new List<SsaChangePointData>();
+            for (int i = 0; i < TrainingSeasons; i++)
+                for (int j = 0; j < SeasonalitySize; j++)
+                    data.Add(new SsaChangePointData(j));
+
+            // Convert data to IDataView.
+            var dataView = ml.CreateStreamingDataView(data);
+
+            // Setup SsaChangePointDetector arguments
+            string outputColumnName = "Prediction";
+            string inputColumnName = "Value";
```

Comment:
>"Value"; [](start = 37, length = 8)

nameof(SsaChangePointData.Value) and same for Prediction as well for both the files.

Reply:
Thanks! Done.

---
In reply to: [238001359](https://github.com/dotnet/machinelearning/pull/1788#discussion_r238001359) [](ancestors = 238001359)